source/overview/features.rst

```diff
@@ -0,0 +1,304 @@
+   :target: ../_images/multi-team_support.png
+
+.. raw:: html
+
+                    <p>Work across multiple teams and projects from the same rich interface.</p>
+                </td>
+                <td>
+                    <h4>Threaded conversations</h4>
+
+.. image:: ../images/threaded_conversations.png
+   :target: ../_images/threaded_conversations.png
+
+.. raw:: html
+
+                    <p>Keep conversations organized and on-point with threading and markdown formatting.</p>
+                </td>
+            <tr>
+                <td>
+                    <h4>Theme colors</h4>
+
+.. image:: ../images/theme_colors.png
+   :target: ../_images/theme_colors.png
+
+.. raw:: html
+
+                    <p>Personalize your experience with build-in and custom themes.</p>
+                </td>
+                <td>
+                    <h4>Multi-language support</h4>
+
+.. image:: ../images/multi-language_support.png
+   :target: ../_images/multi-language_support.png
+
+.. raw:: html
+
+                    <p>Work across borders with one messaging system.</p>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+
+
+
+
+Enterprise Edition Features
+----------------------------------------------
+
+----------------------------------------------
+
+.. raw:: html
+
+    <table class="features__table">
+        <tr>
+            <td>
+                <h4>AD/LDAP SSO</h4>
+
+.. image:: ../images/AD_LDAP_SSO.png
+   :target: ../_images/AD_LDAP_SSO.png
+
+.. raw:: html
+
+                <p>
+                    Streamline user provisioning and permissions with directory service integration.
+                </p>
+            </td>
+            <td>
+                <h4>Advanced permissions</h4>
+
+.. image:: ../images/advanced_permissions.png
+   :target: ../_images/advanced_permissions.png
+
+.. raw:: html
+
+                <p>
+                    Set detailed custom policies and granular permissions for your system.
+                </p>
+            </td>
+        <tr>
+            <td>
+                <h4>MFA</h4>
+
+.. image:: ../images/MFA.png
+   :target: ../_images/MFA.png
+
+.. raw:: html
+
+                <p>
+                    Add an additional layer of security with multi-factor authentication.
+                </p>
+            </td>
+            <td>
+                <h4>Enterprise support</h4>
+
+.. image:: ../images/enterprise_edition_support.png
+   :target: ../_images/enterprise_edition_support.png
+
+.. raw:: html
+
+                <p>
+                    Get access to enterprise-level support to accelerate your deployment and rollout.
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <h4>SAML-based SSO</h4>
+
+.. image:: ../images/SAML_based_SSO.png
+   :target: ../_images/SAML_based_SSO.png
+
+.. raw:: html
+
+                <p>
+                    Enable single sign-on with SAML integration to your directory service.
```

Comment:
"...SAML integration to your directory service." > "...SAML integration added to your directory service." ?